pyls/plugins/flake8_lint.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2017 Palantir Technologies, Inc.
+import logging
+from flake8.api import legacy as flake8
+from pyls import hookimpl, lsp
+
+log = logging.getLogger(__name__)
+
+
```

Comment:
can you add some dev docs, please: what is this, what does it do? please also think about user-facing documentation (@gatesn , where do we keep those?)

Reply:
of course I will